nats.go

```diff
@@ -449,8 +449,9 @@ type Options struct {
 	// away if it can't connect to a server in the initial set. The
 	// MaxReconnect and ReconnectWait options are used for this process,
 	// similarly to when an established connection is disconnected.
-	// If a ReconnectHandler is set, it will be invoked when the connection
-	// is established, and if a ClosedHandler is set, it will be invoked if
+	// If a ReconnectHandler is set, it will be invoked after the first
```

Comment:
```suggestion
	// If a ReconnectHandler is set, it will be invoked on the first
```
More specific on timing?